kustomize/components/memorystore/README.md

```diff
@@ -1,22 +1,22 @@
-# Integrating Online Boutique with Memorystore (redis)
+# Integrate Online Boutique with Memorystore (Redis)
```

Comment:
**Rationale for change:**
[Official Redis docs](https://redis.io/docs/about/) use an uppercase "R".

Reply:
Good point, even in our own Memorystore docs: https://cloud.google.com/memorystore/docs/redis/redis-overview